### DISCUSSION TITLE
Flipello 6x6

### DISCUSSION THREAD

**Discussion by mtaktikos** [fairy-stockfish/Fairy-Stockfish#226] [Variants]
Inspired by alwey's yesterday announced *thello variants (larger board, anti mode)  I want to contribute with the smaller board variant 6x6, that is solved by Feinstein as second player win.

The definition for the variants.ini:
###########
[flipello6]
maxRank = 6
maxFile = 6
immobile = p
startFen = 6/6/2pP2/2Pp2/6/6[PPPPPPPPPPPPPPPPPPPPpppppppppppppppppppp] w 0 1
pieceDrops = true
promotionPieceTypes = -
doubleStep = false
castling = false
stalemateValue = loss
stalematePieceCount = true
materialCounting = unweighted
enclosingDrop = reversi
enclosingDropStart = c3 d3 c4 d4
immobilityIllegal = false
flipEnclosedPieces = reversi
passOnStalemate = true 
##########

Fairy-Stockfish plays this really strong, on 10min/move it gives the evaluation of -1913 (at depth 36 ply), so to say it is "aware" that who begins the game is lost.

**Comment by ianfab** [comment]
Yes, for 6x6 it seems to be close to finding a forced win. For 4x4 it of course is rather trivial, see https://github.com/ianfab/Fairy-Stockfish/wiki/Solved-games#othello-4x4. Just FYI, variant configurations can be simplified quite a bit using inheritance, e.g.

```
[flipello6:flipello]
maxRank = 6
maxFile = 6
startFen = 6/6/2pP2/2Pp2/6/6[PPPPPPPPPPPPPPPPPPPPpppppppppppppppppppp] w 0 1
enclosingDropStart = c3 d3 c4 d4
```
In case both are loaded from an INI it is just important that the inheriting variant is specified after the variant that is inherited from.

**Reply by mtaktikos** [reply]
Hm, the simplified implementation doesn't work.
Although it shows a start-FEN-string like the "full implementation", contrary to this one the simplified prints an empty board, and after the command "go" it doesn't find a move, but declares immediately a draw. See attached png

**Comment by mtaktikos** [comment]
![ProbWithVariantsIni](https://user-images.githubusercontent.com/36267869/103160032-add32e00-47d0-11eb-9e1e-962ae0fba946.png)

**Comment by mtaktikos** [comment]
Problem solved! I used a Fairy-SF exe for large boards, and somehow it had difficulties with the variants.ini. Now with a smaller exe for 8x8 -boards it accepts the simplified implementation, have put the problematic exe in the trash ;)
